src/argilla_sdk/records/_io/_datasets.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2024-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+from typing import Dict, List, Union
+
+from argilla_sdk.records._io._generic import GenericIO
+
+
+def _hf_datasets_installed():
+    """Check if Hugging Face datasets is installed.
+
+    Returns:
+        bool: True if Hugging Face datasets is installed, False otherwise.
+    """
+    try:
+        from datasets import Dataset as HFDataset
+
+        return HFDataset
+    except ImportError:
+        warnings.warn("Hugging Face datasets is not installed. Please install it using `pip install datasets`.")
```

Comment:
With this refactor I think we can skip the warning:
1. When adding records from other than HF datasets, the warning won't be shown (what it makes sense)
2. When exporting as HF datasets without datasets already installed, the flow will raise an error.
```suggestion
```